base/applications/mspaint/dialogs.cpp

```diff
@@ -80,7 +80,10 @@ LRESULT CAttributesDialog::OnInitDialog(UINT nMsg, WPARAM wParam, LPARAM lParam,
     newHeight = imageModel.GetHeight();
 
     CheckDlgButton(IDD_ATTRIBUTESRB3, BST_CHECKED);
-    CheckDlgButton(IDD_ATTRIBUTESRB5, BST_CHECKED);
+    if (!paletteWindow.bk)
+        CheckDlgButton(IDD_ATTRIBUTESRB5, BST_CHECKED);
+    else
+        CheckDlgButton(IDD_ATTRIBUTESRB4, BST_CHECKED);
     SetDlgItemInt(IDD_ATTRIBUTESEDIT1, newWidth, FALSE);
     SetDlgItemInt(IDD_ATTRIBUTESEDIT2, newHeight, FALSE);
```

Comment:
All these lines are 100% the same as `CAttributesDialog::OnDefault()`; it may be better to just call this function here.

Reply:
How I can call OnDefault()? There are no definition of that function, without parameters.

Reply:
Hm yeah it's not terrible... But setting up the default values (on initialization and when pressing a "reset defaults" button) should not be duplicated.